src/plugins/vctlGit.nim

```diff
@@ -8,18 +8,31 @@
 ## The plugin responsible for pulling metadata from the git
 ## repository.
 
-import ../config, ../plugin_api
+import ../config, ../plugin_api, times, zippy, zippy/inflate
 
 const
+  eBadGitConf  = "Git configuration file is invalid"
+  fanoutTable  = 8
+  fanoutSize   = (256 * 4)
   fNameHead    = "HEAD"
   fNameConfig  = "config"
+  highBit32    = uint64(0x80000000)
   ghRef        = "ref:"
   ghBranch     = "branch"
   ghRemote     = "remote"
   ghUrl        = "url"
   ghOrigin     = "origin"
   ghLocal      = "local"
-  eBadGitConf  = "Git configuration file is invalid"
+  gitAuthor    = "author "
+  gitCommitter = "committer "
+  gitIdxAll    = "*.idx"
+  gitIdxExt    = ".idx"
+  gitIdxHeader = "\xff\x7f\x4f\x63\x00\x00\x00\x02"
+  gitObjects   = "objects" & DirSep
+  gitPack      = gitObjects.joinPath("pack")
+  gitPackExt   = ".pack"
```

Comment:
Also unused.

Reply:
the top 3 are used, and I intended to use the 4th, I'll fix these

Reply:
Ahh I think you only meant to point out the bottom one but it includes the lines above it for context (same as your other comment on unused, where the three above it are used)